projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -248,15 +248,15 @@ selectTotal.release();
 
 ## Using Store Without Type Generic
 
-When injecting `Store` into components and other injectables, it is possible to omit the generic type. If injected without the generic, the default generic is applied as follows `Store<T = object>`.
+The most common way to select information from the store is to use a selector function defined with `createSelector`. TypeScript is able to automatically infer types from `createSelector`, which reduces the need to provide the shape of the state to `Store` via a generic argument.
 
-The most common way to select information from the store is to use a selector function defined with `createSelector`. When doing so, TypeScript is able to automatically infer types from the selector function, therefore reducing the need to define the type in the store generic.
+So, when injecting `Store` into components and other injectables, the generic type can be omitted. If injected without the generic, the default generic applied is `Store<T = object>`. 
```

Comment:
- removed use of passive voice
- placed motivation to use `Store` without type generic at beginning of section